python/ray/rllib/agents/ppo/appo_policy.py

```diff
@@ -163,41 +159,6 @@ def __init__(self,
                            self.entropy * entropy_coeff)
 
 
-def _make_time_major(policy, tensor, drop_last=False):
```

Comment:
It looks like the functions in this file was moved to vtrace_policy.py without change, is that right?

Reply:
Yep